mxcubeweb/core/server/resource_handler.py

```diff
@@ -355,22 +357,30 @@ def _validate_param_data(
         # If it's a Pydantic model, validate it
         if issubclass(param_type, BaseModel):
             try:
+                # We only handle a single argument when using Pydantic models
```

Comment:
We only handle a single PyDantic model that model the entire input. However one can pass any number of float, int or str arguments